xml/System/String.xml

```diff
@@ -14710,12 +14710,12 @@ The `TrimStart(System.Char[])` method removes from the current string all leadin
 
 The following example demonstrates the basic functionality of the <xref:System.String.TrimStart%2A> method:
 
-[!code-csharp[System.String.TrimStart#1](~/samples/snippets/csharp/VS_Snippets_CLR_System/system.String.TrimStart/cs/sample.cs#1)]
+[!code-csharp-interactive[System.String.TrimStart#1](~/samples/snippets/csharp/VS_Snippets_CLR_System/system.String.TrimStart/cs/sample.cs#1)]
```

Comment:
there's a bug with try.net with this one

```suggestion
[!code-csharp[System.String.TrimStart#1](~/samples/snippets/csharp/VS_Snippets_CLR_System/system.String.TrimStart/cs/sample.cs#1)]
```

Reply:
Done.